gslb/nodes/node_utils.go

```diff
@@ -224,7 +229,8 @@ func GetDescriptionForPathHMName(hmName string, gsMeta *AviGSObjectGraph) string
 
 func GetPathFromHmDescription(hmName, hmDescription string) string {
 	hmDescriptionSplit := strings.Split(hmDescription, ": ")
-	if len(hmDescriptionSplit) != 5 {
+	if len(hmDescriptionSplit) != 5 &&
+		len(hmDescriptionSplit) != 6 {
```

Comment:
This warning will not be encounter in the case where we expected a length of 5 but got a length of 6 and vice versa

Reply:
The values that we are interested in are the paths and they will be present in the case of lengths 5 or 6. I think it won't cause any issues as the hm path will be present in the 3rd index in both cases.

Reply:
This is true for the encoded HM name change. But in the older(1.5.1 and older) AMKO versions, the HM name is not encode and the description in this case is just `createdby: amko`. For customers transitioning from the say 1.5.1 to 1.7.1, if there are any issues, this warning might never be logged leading to index out of range errors